gammapy/spectrum/models.py

```diff
@@ -1854,3 +1862,23 @@ def evaluate(energy, norm, mean, sigma):
             / (energy * sigma * np.sqrt(2 * np.pi))
             * np.exp(-(np.log(energy / mean)) ** 2 / (2 * sigma ** 2))
         )
+
+
+spectral_models = {           
+            "ConstantModel": ConstantModel,
+            "PowerLaw": PowerLaw,
+            "PowerLaw2": PowerLaw2,
+            "ExponentialCutoffPowerLaw": ExponentialCutoffPowerLaw,
+            "ExponentialCutoffPowerLaw3FGL": ExponentialCutoffPowerLaw3FGL,
+            "PLSuperExpCutoff3FGL": PLSuperExpCutoff3FGL,
+            "PLSuperExpCutoff4FGL": PLSuperExpCutoff4FGL,
+            "LogParabola": LogParabola,
+            "TableModel": TableModel,
+            "SpectralGaussian": SpectralGaussian,
+            "SpectralLogGaussian": SpectralLogGaussian,
+            "ScaleModel": ScaleModel,
+#               TODO: add support for these models writing their .from_dict()
```

Comment:
Maybe `make black` doesn't format that dict properly because of the inline comment?
Suggest to move the comment before or after the dict.